<launch>
  <param name="ec_cluster_tolerance" value="0.03" />
  <param name="ec_min_cluster_size" value="700" />
  <param name="ec_max_cluster_size" value="7500" />
  <node pkg="perception" type="point_cloud_demo" name="point_cloud_demo" output="screen">
    <remap from="cloud_in" to="mock_point_cloud" />
  </node>
</launch>